frontend-react/src/components/header/BasicHelmet.test.tsx

```diff
@@ -1,4 +1,6 @@
-import { render, waitFor } from "@testing-library/react";
```

Comment:
This existed beforehand so I'm not commenting on the state of your pull request, but I don't really understand the purpose of this test.  Seems like we're testing basic functionality of React and react-helmet(-async) functionality, which isn't really in our purview of maintenance 🤔 

Like if either React or Helmet breaks and doesn't render this component correctly, that's not something we can fix 🤷 🤷 🤷

Reply:
I would go as far as saying I don't understand the purpose of the BaseHelmet at all. You're not saving worthwhile time/kb on it currently vs just calling Helmet directly with a title.

Reply:
💯 💯 💯 💯 💯 💯 💯 

Want to add a fast-follow cleanup ticket to remove it?  😈